cypress/e2e/cloud/globalHeaderOrgMenuItems.test.ts

```diff
@@ -0,0 +1,81 @@
+import {makeQuartzUseIDPEOrgID} from 'cypress/support/Utils'
+
+const globalHeaderFeatureFlags = {
+  quartzIdentity: true,
```

Comment:
`quartzIdentity` flag is gone, so okay to remove this throughout.

Reply:
i'll remove from the 2 test files. We should clean up this flag in another issue. Created: https://github.com/influxdata/ui/issues/5997

Reply:
It has already been removed everywhere else